sdk/confidentialledger/Azure.Security.ConfidentialLedger/README.md

```diff
@@ -72,21 +72,15 @@ Since Azure confidential ledger is a distributed system, rare transient failures
 ```C# Snippet:GetStatus
 Response statusResponse = ledgerClient.GetTransactionStatus(transactionId);
 
-string status = JsonDocument.Parse(statusResponse.Content)
-    .RootElement
-    .GetProperty("state")
-    .GetString();
+string status = statusResponse.Content.ToDynamic().state;
```

Comment:
I think out samples should always store the created DynamicJson in a variable, even if the sample uses the variable once. Otherwise, I worry that users will be reparsing the JSON many times as they evolve the code. i.e. instead of the line above, let's use:

```csharp
dynamic content = statusResponse.Content.ToDynamic();
string status = content.state;
```

Reply:
Cool, I was looking to minimize changes to the sample, but happy to set this as a best practice.  Thanks!

Reply:
I don't think we should standardize on naming the variable `content` as it is generic to the name of the response payload property. I'd suggest we pick a name related to the type being returned. e.g.

```c#
dynamic transactionStatus = statusResponse.Content.ToDynamic();
string status = transactionStatus.state;
```

Reply:
I think the name of the dynamic variable is less important than the fact that we call ToDynamic() separately and then access members on it in separate lines of code.  Variable naming can be left to the author's discretion.

Reply:
Sure, I'm happy to use the name you suggested for this variable when I loop back and make the suggested change.